Adjust test cases to work with busybox.

- Replace dd parameter "obs" with "bs".
- Replace "head -<num>" with "head -n <num>".

Signed-off-by: Maxin B. John <maxin.john@enea.com>
Upstream-Status: Pending

Index: openssh-7.6p1/regress/cipher-speed.sh
===================================================================
--- openssh-7.6p1.orig/regress/cipher-speed.sh
+++ openssh-7.6p1/regress/cipher-speed.sh
@@ -17,7 +17,7 @@ for c in `${SSH} -Q cipher`; do n=0; for
 		printf "%-60s" "$c/$m:"
 		( ${SSH} -o 'compression no' \
 			-F $OBJ/ssh_proxy -m $m -c $c somehost \
-			exec sh -c \'"dd of=/dev/null obs=32k"\' \
+			exec sh -c \'"dd of=/dev/null bs=32k"\' \
 		< ${DATA} ) 2>&1 | getbytes
 
 		if [ $? -ne 0 ]; then
Index: openssh-7.6p1/regress/transfer.sh
===================================================================
--- openssh-7.6p1.orig/regress/transfer.sh
+++ openssh-7.6p1/regress/transfer.sh
@@ -13,7 +13,7 @@ cmp ${DATA} ${COPY}		|| fail "corrupted
 for s in 10 100 1k 32k 64k 128k 256k; do
 	trace "dd-size ${s}"
 	rm -f ${COPY}
-	dd if=$DATA obs=${s} 2> /dev/null | \
+	dd if=$DATA bs=${s} 2> /dev/null | \
 		${SSH} -q -F $OBJ/ssh_proxy somehost "cat > ${COPY}"
 	if [ $? -ne 0 ]; then
 		fail "ssh cat $DATA failed"
Index: openssh-7.6p1/regress/key-options.sh
===================================================================
--- openssh-7.6p1.orig/regress/key-options.sh
+++ openssh-7.6p1/regress/key-options.sh
@@ -47,7 +47,7 @@ for f in 127.0.0.1 '127.0.0.0\/8'; do
 	fi
 
 	sed 's/.*/from="'"$f"'" &/' $origkeys >$authkeys
-	from=`head -1 $authkeys | cut -f1 -d ' '`
+	from=`head -n 1 $authkeys | cut -f1 -d ' '`
 	verbose "key option $from"
 	r=`${SSH} -q -F $OBJ/ssh_proxy somehost 'echo true'`
 	if [ "$r" = "true" ]; then
